react/features/breakout-rooms/actions.js

```diff
@@ -145,9 +145,10 @@ export function autoAssignToBreakoutRooms() {
  *
  * @param {string} participantId - The participant id.
  * @param {string} roomId - The room id.
+ * @param {boolean} isJid - Whether the participantId is a jid.
```

Comment:
Why can't we detect this within the function itself? In fact we do something like that inside `_findParticipantJid`